### PR TITLE
Improvements to Teams documentation

### PR DESCRIPTION
**Pull request by keon94**

### CODE REVIEW

File: docs/UserManuals/TeamConfiguration.md [hunk 1]
```diff
@@ -52,19 +52,19 @@ b. The actual api request.
...
     
     iv. Generated the team_users table, you can see the data in the team_users table.
 
-![image](/img/Team/teamflow2.png)
+![image](../../static/img/Team/teamflow2.png)
 
 ## Step 3 - Update users if you need  
 If there is a problem with team_users association or data in users, just re-put users api interface, i.e. (b in step 2 above)
 
 ## Step 4 - Collect accounts
```
Comment (keon94) [inline docs/UserManuals/TeamConfiguration.md]: @hezyin I think we need to be more clear here. This guide generates dummy data for demoing purposes, but this step is requiring actual data to be present in the accounts table. Even if we run some plugin execution, that's not going to be generating account data that would correspond to the dummy user data. As a result, the user (the person running the demo) technically has to go in the accounts table and manually add rows that'll correspond to the dummy users, in order to make steps 5-7 be able to fetch user-account associations. I think this is a gap we need to fill in the doc.

**Comment by yumengwang03** [comment]
Hi Keon,

The image URL should still be "/img/xxx.png" because Docusaurus (the tool we are using to build the website) recognize this path as the same as "../../static/img/xxx.png". We also need to use "/img/xxx.png" because otherwise the versioned docs will contain wrong paths to the images :) I'll put these details into a guide for submitting docs to the website.

File: docs/UserManuals/TeamConfiguration.md (hunk 1 above)
Comment (abeizn) [reply docs/UserManuals/TeamConfiguration.md]: Thank you, keon. Very good advice, I will update the documentation.

Comment (abeizn) [reply docs/UserManuals/TeamConfiguration.md]: updated: https://github.com/apache/incubator-devlake-website/pull/120/files

**Comment by keon94** [comment]
> Hi Keon,
> 
> The image URL should still be "/img/xxx.png" because Docusaurus (the tool we are using to build the website) recognize this path as the same as "../../static/img/xxx.png". We also need to use "/img/xxx.png" because otherwise the versioned docs will contain wrong paths to the images :) I'll put these details into a guide for submitting docs to the website.

Okay, thanks for letting me know. I've reverted the change.

**Review by hezyin — APPROVED** [review]
LGTM

**Pull request merged**